src/commands/apextests/list.ts

```diff
@@ -36,13 +36,11 @@ export default class ApextestsList extends SfCommand<ApextestsListResult> {
     // remove the prefix @Tests or @TestSuites
     return data
       .split(',')
-      .map((line) => line.replace(/(@Tests|@TestSuites)/, ''))
-      .map((line) => line.replace(':', ''))
-      .map((line) => line.trim())
-      .filter((line) => line.trim().length > 0);
+      .map((line) => line.replace(/(@Tests|@TestSuites)/, '').replace(':', ''))
+      .map((line) => line.trim());
   }
 
-  private static listTestsInDirectory(directory: string): string[] {
+  private static async listTestsInDirectory(directory: string): Promise<string[]> {
```

Comment:
**remarks:** personal note, I do not like the static for this kind of situation, it is harder to mock and test then

Reply:
I'm indifferent regarding these being static or not, but in this case the choice for static is just because of the linter. It complains about a method not using `this` anywhere and then suggests making it a static method. Do you think this could be an obstacle in the future?